Source/com/CMakeLists.txt

```diff
@@ -18,17 +18,11 @@
 set(TARGET ${NAMESPACE}COM)
 
 ProxyStubGenerator(NAMESPACE "WPEFramework::RPC" INPUT "${CMAKE_CURRENT_SOURCE_DIR}/ICOM.h" OUTDIR "${CMAKE_CURRENT_BINARY_DIR}/generated")
-
-set(PROXYSTUB_COM_SOURCE "${CMAKE_CURRENT_BINARY_DIR}/generated/ProxyStubs_COM.cpp")
-
-configure_file( "${CMAKE_CURRENT_LIST_DIR}/ProxyStubs.cpp.in"
-                "${CMAKE_CURRENT_BINARY_DIR}/ProxyStubs.cpp"
-                @ONLY)
+ProxyStubGenerator(NAMESPACE "WPEFramework::Trace" INPUT "${CMAKE_CURRENT_SOURCE_DIR}/ITrace.h" OUTDIR "${CMAKE_CURRENT_BINARY_DIR}/generated")
 
 add_library(${TARGET} SHARED
         Administrator.cpp
         Communicator.cpp
-        "${CMAKE_CURRENT_BINARY_DIR}/ProxyStubs.cpp"
         ITracing.cpp
```

Comment:
Does it require ProxyStubs_* files here?

Reply:
I think you have identified an issue. The ITrace proxy/stub should indeed be coming from the com module. (it is part of it, like IUnknown proxy/stub).
However, mke sure we can slos still build the com and websocket projects seperatly (some ustomers do so). I saw you removed the ProxyStub.in.cpp in com??? don't know if it is causing an issue, just a remark :-)

Reply:
yes, agree :-). This is required to create COM libs. thats why I did not notice it during the build